clarifai/client/model.py

```diff
@@ -1295,15 +1296,22 @@ def _download_exported_model(
       model_export_url = get_model_export_response.export.url
       model_export_file_size = get_model_export_response.export.size
 
-      response = requests.get(model_export_url, stream=True)
+      session = requests.Session()
+      retries = Retry(total=10, backoff_factor=1, status_forcelist=[500, 502, 503, 504])
+      session.mount('https://', HTTPAdapter(max_retries=retries))
+      session.headers.update({'Authorization': self.metadata[0][1]})
+      response = session.get(model_export_url, stream=True)
       response.raise_for_status()
 
       with open(local_filepath, 'wb') as f:
         progress = tqdm(
             total=model_export_file_size, unit='B', unit_scale=True, desc="Exporting model")
         for chunk in response.iter_content(chunk_size=8192):
-          f.write(chunk)
-          progress.update(len(chunk))
+          if chunk:
```

Comment:
what is the intention behind this if else block?
We are not expecting chunk to be empty or None ever. We can undo this change.

Reply:
Intially, i thought there is some issue with the chucks, so I have added this if else block.I will undo this change now.

Reply:
Done